.github/workflows/ps-ce-sdk-release.yaml

```diff
@@ -0,0 +1,44 @@
+# **************************************************************************
+#  Copyright (c) Cloud Native Foundation.
+#  SPDX-License-Identifier: Apache-2.0
+# **************************************************************************
+
+name: Release
+
+on:
+  create:
+    tags:
+      - v*
+
+jobs:
+  psgallery-publish:
+    name: Release CloudEvents.Sdk Module
+    runs-on: "ubuntu-latest"
+
+    timeout-minutes: 10
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Build and Test
+        shell: pwsh
+        run: ./build.ps1 -OutputDir 'release' -TestsType all -ExitProcess
```

Comment:
nit: 'release' could be made job-level env var since it's reused and allows for easy changes later